mm/2s2h/Rando/ActorBehavior/EnemyDrops.cpp

```diff
@@ -101,17 +100,24 @@ void Rando::ActorBehavior::InitEnemyDropBehavior() {
             return;
         }
 
-        Actor* foundActor = FindActor(position);
+        Actor* foundActor = FindActor(position, ACTORCAT_ENEMY);
         if (foundActor == nullptr || foundActor->category != ACTORCAT_ENEMY) {
             return;
         }
 
-        if (foundActor->id == ACTOR_EN_SW) {
+        int16_t actorId = foundActor->id;
+        // Ghibdo ID's are different for Patroling Music House and Beneath the Well
+        if (actorId == ACTOR_EN_RAIL_SKB || actorId == ACTOR_EN_HINT_SKB) {
+            actorId = ACTOR_EN_RD;
```

Comment:
Your comment and `ACTOR_EN_RD` refer to Gibdos, but `ACTOR_EN_RAIL_SKB` and `ACTOR_EN_HINT_SKB` are for Stalchildren. I'm fine with this approach, but this conversion crosses wires. There should be two cases:

- `ACTOR_EN_TALK_GIBUD` and `ACTOR_EN_RAILGIBUD` convert to `ACTOR_EN_RD`
- `ACTOR_EN_RAIL_SKB` and `ACTOR_EN_HINT_SKB` convert to `ACTOR_EN_SKB`
```suggestion
        // Handle enemy types that have multiple different actor types
        switch (actorId) {
            case ACTOR_EN_TALK_GIBUD:
            case ACTOR_EN_RAILGIBUD:
                actorId = ACTOR_EN_RD; // Gibdos
                break;
            case ACTOR_EN_RAIL_SKB:
            case ACTOR_EN_HINT_SKB:
                actorId = ACTOR_EN_SKB; // Stalchildren
                break;
            default:
                break;
        }
```